File: src/main/java/com/axodouble/modules/arena/ArenaModule.java
package com.axodouble.modules.arena;

import com.axodouble.Double;

public class ArenaModule {
    private Double plugin;
    public ArenaManager arenaManager;
    public ArenaModule(Double plugin) {
        this.plugin = plugin;
        new ArenaCommandHandler(plugin);
        new ArenaEvents(plugin);
        new ArenaActions(plugin);
        arenaManager = new ArenaManager(plugin);
    }
}

File: src/main/java/com/axodouble/modules/arena/ArenaModule.kt
package com.axodouble.modules.arena

import com.axodouble.Ceraia

class ArenaModule(private val plugin: Ceraia) {
    @JvmField
    var arenaManager: ArenaManager
    var arenaEvents: ArenaEvents
    var arenaActions: ArenaActions

    init {
        ArenaCommandHandler(plugin)
        arenaEvents = ArenaEvents(plugin)
        arenaActions=ArenaActions(plugin)
        arenaManager = ArenaManager(plugin)
    }
}
